styles.css

```
@@ -1,13 +1,44 @@
 body {
-    padding: 0 200px;
+  padding: 0;
+  margin: 0;
+  font-family: Roboto;
 }
 /*navbar*/
 
 /*header*/
         
 /*section1*/
-       
-/*section2*/
+.testing{
+  border: red solid 1px;
+}
+.imageRow{
+  padding: 0 400px;
```

Please fix the top and bottom padding

```suggestion
  padding: 50px 400px;
```